python/.venv/bin/Activate.ps1

```diff
@@ -0,0 +1,247 @@
+<#
```

Comment:
Everything under .venv shouldn't be checked in as it's a virtualenv that's specific to your local path and not transferable to other location, and hence will break unless they have the exact configuration as you.

Please remove the entire .venv dir and ideally rebase so the commit with the files is removed from commit history (though I think it will still be something in the repo just hidden).  
It is recommended to add .venv to .gitignore to avoid checking this in.